client/rest-high-level/src/test/java/org/elasticsearch/client/documentation/QueryDSLDocumentationTests.java

```diff
@@ -189,7 +189,7 @@ public void testGeoShape() throws IOException {
             // tag::geo_shape
             GeoShapeQueryBuilder qb = geoShapeQuery(
                     "pin.location",                                      // <1>
-                    ShapeBuilders.newMultiPoint(                         // <2>
+                    new MultiPointBuilder(                         // <2>
```

Comment:
I don't think we can remove the ShapeBuilders helper class in 6.1., that might break some peoples code and I think this is user-facing enough. Not sure if that was the plan, I think its okay to break in 7.0 but then it needs to be deprecated.

Reply:
:+1:  I can keep this PR for 7.0 and open a separate deprecation PR for 6.1?

Reply:
Sound good.